docs/compass/03-packages-credential-requests.md

```diff
@@ -0,0 +1,137 @@
+# Credential requests for Packages
+
+## Overview
+
+On Runtime, Application is represented as Service Class, and every Package within Application is represented as Service Plan. This document describes credential requests for APIs during the Service Plan provisioning. It also mentions passing optional input parameters from Kyma Runtime to Application or Integration System during the provisioning process.
+
+## Assumptions
+
+- Multiple Service Instances can be created from a given Package within Application.
```

Comment:
maybe this will be more informative:
> Multiple Service Instances can be created from a given Package in one Runtime. For example on Kyma Runtime service instance can be created for every namespace.

Can I create 2 Service Instance in the same ns?